metadata_backend/helpers/parser.py

```diff
@@ -125,13 +125,31 @@ def _flatten(self, data: ElementData) -> Union[Dict, List, str, None]:
                     children[key] = value
                 continue
 
-            if "assembly" in key:
-                if next(iter(value)) in {"standard", "custom"}:
-                    children[key] = next(iter(value.values()))
-                    if "accessionId" in children[key]:
-                        children[key]["accession"] = children[key].pop("accessionId")
+            if key in {"processedReads", "referenceAlignment", "sequenceAnnotation", "assemblyAnnotation"}:
+                if next(iter(value)) in {"assembly"}:
+                    # value["assembly"] = {}
+                    if next(iter(value["assembly"])) in {"standard", "custom"}:
+                        value["assembly"] = next(iter(value["assembly"].values()))
+                        # print(children[key])
+                        if "accessionId" in value["assembly"]:
+                            value["assembly"]["accession"] = value["assembly"].pop("accessionId")
+                children[key] = value
+                continue
+
+            if key == "assemblyGraph" and "assembly" in value:
+                attribs = value["assembly"]
+                attr_list = []
+                if isinstance(attribs, list):
+                    for d in attribs:
+                        for k, v in d.items():
+                            v["accession"] = v.pop("accessionId")
+                            attr_list.append(v)
```

Comment:
Surprising that the style tests don't complain about this many nested statements :D

Reply:
yeah, surprised as well, also noticed some comments there, will remove those ...